kong/plugins/prometheus/exporter.lua

```diff
@@ -190,10 +197,26 @@ if kong_subsystem == "http" then
       route_name = message.route.name or message.route.id
     end
 
+    local consumer = ""
+    if message and serialized.consumer ~= nil then
+      consumer = serialized.consumer
+    end
+
     labels_table[1] = service_name
     labels_table[2] = route_name
     labels_table[3] = message.response.status
-    metrics.status:inc(1, labels_table)
+    labels_table[4] = consumer
+
+    labels_table_status[1] = service_name
+    labels_table_status[2] = route_name
+    labels_table_status[3] = message.response.status
+    labels_table_status[4] = kong.response.get_source()
+    labels_table_status[5] = consumer
```

Comment:
as stream doesn't have consumer, you might want to set it to `nil` instead of passing a `""`, otherwise the prometheu library will cry

Reply:
this is for http subsystem.